test/e2e/examples_test.go

```diff
@@ -98,3 +98,41 @@ func TestRustGameServerReady(t *testing.T) {
 	// Assert that the GameServer is in the expected state
 	assert.Equal(t, agonesv1.GameServerStateReady, readyGs.Status.State)
 }
+
+func TestNodeJSGameServerReady(t *testing.T) {
+	t.Parallel()
+	gs := &agonesv1.GameServer{
+		ObjectMeta: metav1.ObjectMeta{
+			GenerateName: "nodejs-simple-",
+		},
+		Spec: agonesv1.GameServerSpec{
+			Ports: []agonesv1.GameServerPort{{
+				Name:          "default",
+				PortPolicy:    agonesv1.Dynamic,
+				ContainerPort: 7654,
+				Protocol:      corev1.ProtocolUDP,
+			}},
+			Health: agonesv1.Health{
+				InitialDelaySeconds: 30,
```

Comment:
Let's make `initialDelaySeconds` bigger 😃 I think image pulldown in the cluster that has a lot of traffic is taking longer than expected.

Let's try `300` seconds. 5 minutes.

Reply:
Sure, will change it. Any changes required for `PeriodSeconds`?

Reply:
Nah I don't think so. My theory is that for some reason it's taking a while to get up and running. Once it's up and running period seconds should be fine even if it's far lower.